samples/csharp/MicrosoftEduGraphSamples/appsettings.json

```diff
@@ -5,5 +5,7 @@
   "classId": "YOUR_CLASS_ID",
   "teacherAccount": "YOUR_TEACHER_ACCOUNT",
   "studentAccount": "YOUR_STUDENT_ACCOUNT",
-  "password": "YOUR_PASSWORD"
+  "teacherpassword": "YOUR_PASSWORD",
+  "studentpassword": "YOUR_PASSWORD"
```

Comment:
it seems to be a breaking change. You need to update all workflows and test to make sure they are working fine.

Reply:
verified for all workflows